src/settings-ui/Settings.UI/SettingsXAML/OOBE/Views/OobeWhatsNew.xaml.cs

```diff
@@ -248,5 +246,11 @@ private void DataDiagnostics_OpenSettings_Click(Microsoft.UI.Xaml.Documents.Hype
         {
             Common.UI.SettingsDeepLink.OpenSettings(Common.UI.SettingsDeepLink.SettingsWindow.Overview, true);
         }
+
+        private void SetInfoBar(InfoBar infoBar, bool open)
```

Comment:
Honestly, you could do without this new function. Setting the **IsOpen** property directly is fine. If you want to keep this, then I suggest a name like `SetInfoBarIsOpen` to be specific.

Reply:
Indeed, `IsTabStop` is not needed. Changed.